tests/test_config_utils.py

```diff
@@ -31,21 +30,6 @@ def test_bigchain_instance_is_initialized_when_conf_provided(request):
     assert bigchaindb.config['CONFIGURED'] is True
 
 
-def test_bigchain_instance_raises_when_not_configured(request, monkeypatch):
```

Comment:
I couldn't understand why don't we need this test anymore?

Reply:
Where do you expect `ConfigurationError` to come from? From the first glance, without a config assigned, it will fail with a different exception. Also, the test setup assigns a config so it does not work either way.

Reply:
It seems that this test wasn't of much help before. I guess  we can do without it then

Reply:
It seems like the intent of the test is to make sure we fail in a controlled fashion if the config gets dropped in some way. Couldn't we modify it to use a patched `autoconfigure` with `force=True` to set a bad config and produce the error that way?

Reply:
Currently, the function does not raise `ConfigurationError` in case the config is not set or empty. The contract that was tested here is lost.